go.mod

```diff
@@ -7,15 +7,15 @@ require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/evanphx/json-patch v4.5.0+incompatible
 	github.com/golang/mock v1.4.3
-	github.com/golang/protobuf v1.3.5
+	github.com/golang/protobuf v1.4.2
 	github.com/imdario/mergo v0.3.9 // indirect
 	github.com/kubernetes-csi/csi-lib-utils v0.7.0
 	github.com/kubernetes-csi/csi-test/v3 v3.1.0
 	google.golang.org/grpc v1.28.0
 	k8s.io/api v0.18.0
 	k8s.io/apimachinery v0.18.0
 	k8s.io/client-go v0.18.0
-	k8s.io/csi-translation-lib v0.18.0
+	k8s.io/csi-translation-lib v0.0.0-20200508210126-48562615b4e9
```

Comment:
Can you provide a link to this hash?

Reply:
changed tag to v0.19.0-rc.2 for k8s.io/csi-translation-lib.
v0.19.0-rc.2 includes vSphere CSI Migration support.